internal/provider/restore_resource.go

```diff
@@ -0,0 +1,403 @@
+						Optional:    true,
+					},
+					"into_db": schema.StringAttribute{
+						Description: "Specifies the target database to restore the table into during a table restore operation. If not set, the table will be restored into the database it belonged to in the source backup.",
+						Optional:    true,
+					},
+					"skip_localities_check": schema.BoolAttribute{
+						Description: "Allows the restore operation to continue in the event that there are mismatched localities between the backup and target cluster. Useful when restoring multi-region tables to a cluster missing some localities.",
+						Optional:    true,
+					},
+					"skip_missing_foreign_keys": schema.BoolAttribute{
+						Description: "Allows a table to be restored even if it has foreign key constraints referencing rows that no longer exist in the target cluster.",
+						Optional:    true,
+					},
+					"skip_missing_sequences": schema.BoolAttribute{
+						MarkdownDescription: "Allows a table to be restored even if it contains a column whose `DEFAULT` value depends on a sequence. More information can be found [here](https://www.cockroachlabs.com/docs/stable/show-sequences).",
+						Optional:            true,
+					},
+					"schema_only": schema.BoolAttribute{
+						Description: "If set, only the schema will be restored and no user data will be included.",
+						Optional:    true,
+					},
+				},
+			},
+			"status": schema.StringAttribute{
+				Description: "The current status of the restore operation.",
+				Computed:    true,
+			},
+			"created_at": schema.StringAttribute{
+				Description: "The time at which the restore operation was initiated.",
+				Computed:    true,
+			},
+			"completion_percent": schema.Float32Attribute{
+				Description: "The percentage of the restore operation that has been completed. The value ranges from 0 to 1.",
+				Computed:    true,
+			},
+		},
+		Blocks: map[string]schema.Block{
+			"objects": schema.ListNestedBlock{
+				Description: "The list of objects to restore. Required when type is DATABASE or TABLE.",
+				NestedObject: schema.NestedBlockObject{
+					Attributes: map[string]schema.Attribute{
+						"database": schema.StringAttribute{
+							MarkdownDescription: "The database portion of the name of the objects to be restored. In the case of `tpcc.public.warehouse`, this would be `tpcc`.",
+							Required:            true,
+						},
+						"schema": schema.StringAttribute{
+							MarkdownDescription: "The schema portion of the name of the objects to be restored. In the case of `tpcc.public.warehouse`, this would be `public`.",
+							Optional:            true,
+						},
+						"tables": schema.ListAttribute{
+							MarkdownDescription: "The table portion of the name of the objects to be restored. In the case of `tpcc.public.warehouse`, this would be `warehouse`.",
```

Comment:
```suggestion
							MarkdownDescription: "The table name in the fully qualified name of the objects to be restored. In the case of an operation that restores `tpcc.public.warehouse`, this would value be `warehouse`.",
```